environments/main/environment.yml

```diff
@@ -71,8 +76,4 @@ dependencies:
 
   # Pip dependencies could be imported or non-imported :(
   - pip:
-    - sphinx-rtd-theme ~=1.0.0
-    - autodoc-pydantic ~=1.5.1
     - sphinx-selective-exclude ~=1.0.3
-    # TODO: get this dependency from conda-forge (see https://github.com/callowayproject/bump-my-version/issues/46)
-    - bump-my-version ~=0.9.2
```

Comment:
All but one is now on conda-forge. I was hoping to get rid of this whole section!